Library/ExternalSubscription.cs

```diff
@@ -0,0 +1,189 @@
+                            LastPurchased = dateVal;
+                        break;
+
+                    case "auto_renew":
+                        bool b;
+                        if (bool.TryParse(reader.ReadElementContentAsString(), out b))
+                            AutoRenew = b;
+                        break;
+
+                    case "app_identifier":
+                        AppIdentifier = reader.ReadElementContentAsString();
+                        break;
+
+                    case "quantity":
+                        Quantity = reader.ReadElementContentAsInt();
+                        break;
+
+                    case "activated_at":
+                        if (DateTime.TryParse(reader.ReadElementContentAsString(), out dateVal))
+                            ActivatedAt = dateVal;
+                        break;
+
+                    case "expires_at":
+                        if (DateTime.TryParse(reader.ReadElementContentAsString(), out dateVal))
+                            ExpiresAt = dateVal;
+                        break;
+
+                    case "updated_at":
+                        if (DateTime.TryParse(reader.ReadElementContentAsString(), out dateVal))
+                            UpdatedAt = dateVal; ;
+                        break;
+
+                    case "created_at":
+                        if (DateTime.TryParse(reader.ReadElementContentAsString(), out dateVal))
+                            CreatedAt = dateVal; ;
+                        break;
+                }
+            }
+        }
+
+        internal override void WriteXml(XmlTextWriter writer)
+        {
+            throw new NotImplementedException();
+        }
+
+        #endregion
+    }
+
+    public sealed class ExternalSubscriptions
+    {
+        /// <summary>
+        /// Returns a list of recurly external_subscriptions
+        /// </summary>
+        /// <returns></returns>
+        public static RecurlyList<ExternalSubscription> List()
+        {
+            return List(null);
+        }
+
+        /// <summary>
+        /// Returns a list of recurly external_products
+        ///
+        /// A external_product will belong to more than one state.
+        /// </summary>
+        /// <param name="filter">FilterCriteria used to apply server side sorting and filtering</param>
+        /// <returns></returns>
+        public static RecurlyList<ExternalSubscription> List(FilterCriteria filter)
+        {
+            filter = filter ?? FilterCriteria.Instance;
+            var parameters = filter.ToNamedValueCollection();
+            return new ExternalSubscriptionList(ExternalSubscription.UrlPrefix + "?" + parameters.ToString());
+        }
+
+        public static ExternalSubscription Get(string uuid)
+        {
+            if (string.IsNullOrWhiteSpace(uuid))
+            {
+                return null;
+            }
+            var s = new ExternalSubscription();
```

Comment:
I personally don't love single letters as variable names, are you open to changing this `external_sub`